### PR TITLE
Update Openstack installation r2

### PR DESCRIPTION
**Pull request by lapentad**
Renamed the md file.
Deleted extra steps and added single command for install.
Linked to the manual installation of the kpt packages for the common components.

### CODE REVIEW

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+> **_NOTE:_** The Ansible script will try to install some utilities, if for example the script fails to uninstall them first, you can manually delete them and re-run the script.
+```find /usr/lib/python* -type f -name "PyYAML*"``` and ```sudo rm /usr/lib/python3/dist-packages/PyYAML-5.3.1.egg-info```
+
+### Manual Installation of the management cluster using kpt
+
+- [Common Components](/content/en/docs/guides/install-guides/common-components.md)
+- [Common Dependencies](/content/en/docs/guides/install-guides/common-dependencies.md)
+
+## Manual Installation of the Edge cluster using kpt
+Install rootsync CRD from:
```
Comment (efiacor) [inline content/en/docs/guides/install-guides/install-on-openstack.md:49]:
```suggestion
Install config-sync using:

```

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  --namespace default \
+  --repo-basic-username=${GITHUB_USERNAME} \
+  --repo-basic-password=${GITHUB_TOKEN} \
+  --create-branch=true \
+  --deployment=true \
+  <http url of your edge repo>
+```
+
+## Deploy packages to the edge clusters
+Using the web-ui add a new deployment with destination the edge cluster.
```
Comment (efiacor) [inline content/en/docs/guides/install-guides/install-on-openstack.md:94]:
```suggestion
Using the web-ui, add a new deployment to the edge workload cluster.

```

**Review by efiacor — COMMENTED** [review]
Minor comments

File: content/en/docs/guides/install-guides/install-on-openstack.md (hunk 1 above)
Comment (lapentad) [reply content/en/docs/guides/install-guides/install-on-openstack.md:49]: I meant to say *catalog\nephio\core\configsync\rootsync-crd.yaml* inside configsync package

I remember before rootsync-crd was in rootsync package.

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+then run:
+```bash
+wget -qO- https://raw.githubusercontent.com/nephio-project/test-infra/main/e2e/provision/init.sh | \
+sudo \
+NEPHIO_DEBUG=true \
+NEPHIO_BRANCH=main \
+NEPHIO_USER=$USER \
+NEPHIO_RUN_E2E=false \
+ANSIBLE_CMD_EXTRA_VAR_LIST="k8s.context=kubernetes-admin@cluster.local kind.enable=false host_min_vcpu=4 host_min_cpu_ram=8" \
+bash
```
Comment (electrocucaracha) [inline content/en/docs/guides/install-guides/install-on-openstack.md:36]: `NEPHIO_RUN_E2E` is false by [default](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/init.sh#L57)

`NEPHIO_BRANCH` is main by [default](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/init.sh#L59)

I'd suggest to not enable debug for simplicity

```suggestion
export NEPHIO_USER=$USER
export ANSIBLE_CMD_EXTRA_VAR_LIST="k8s.context=kubernetes-admin@cluster.local kind.enable=false host_min_vcpu=4 host_min_cpu_ram=8"

curl -fsSL https://raw.githubusercontent.com/nephio-project/test-infra/main/e2e/provision/init.sh | sudo -E bash
```

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+```
+- - - -
+
+> **_NOTE:_** The Ansible script will try to install some utilities, if for example the script fails to uninstall them first, you can manually delete them and re-run the script.
+```find /usr/lib/python* -type f -name "PyYAML*"``` and ```sudo rm /usr/lib/python3/dist-packages/PyYAML-5.3.1.egg-info```
```
Comment (electrocucaracha) [inline content/en/docs/guides/install-guides/install-on-openstack.md:41]: These instructions need to be included in test infra, maybe somewhere around [here](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/install_sandbox.sh#L25)

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+title: OpenStack deployment management
```
Comment (electrocucaracha) [inline content/en/docs/guides/install-guides/install-on-openstack.md:2]:
```suggestion
title: Nephio manual deployment on multiple VMs
```

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
+---
+title: OpenStack deployment management
+description: >
+  OpenStack Multi-Cluster Deployment Management with Nephio
```
Comment (electrocucaracha) [inline content/en/docs/guides/install-guides/install-on-openstack.md:4]:
```suggestion
  Nephio manual deployment on different VMs
```

**Review by electrocucaracha — COMMENTED** [review]
@lapentad I realized that these instructions can be applied to any setup provisioned by any Cloud provider, so it's not only applicable for OpenStack.

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+description: >
+  Nephio manual deployment on different VMs
+weight: 7
+---
+
+## Prerequisites
+
+* OpenStack Cluster Management (master)
+
+  * 4 vCPU
```
Comment (kushnaidu) [inline content/en/docs/guides/install-guides/install-on-openstack.md:12]:
```suggestion
* OpenStack Cluster Management (master)
  * 4 vCPU
  * 8 GB RAM
  * Kubernetes version 1.26+

* OpenStack Cluster Edge n
  * 2 vCPU 1 NODE
  * 4 GB RAM
  * Kubernetes version 1.26+
  
* KPT beta [releases](https://github.com/kptdev/kpt/releases)
```

File: content/en/docs/guides/install-guides/install-on-openstack.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+  * 8 GB RAM
+
+  * Kubernetes version 1.26+
+* OpenStack Cluster Edge n
+
+  * 2 vCPU 1 NODE
+  * 4 GB RAM
+  * Kubernetes version 1.26+
+* KPT beta [releases](https://github.com/kptdev/kpt/releases)
```
Comment (kushnaidu) [inline content/en/docs/guides/install-guides/install-on-openstack.md:21]:
```suggestion
* OpenStack Cluster Management (master)

  * 4 vCPU
  * 8 GB RAM
  * Kubernetes version 1.26+
  
* OpenStack Cluster Edge n

  * 2 vCPU 1 NODE
  * 4 GB RAM
  * Kubernetes version 1.26+

* KPT beta [releases](https://github.com/kptdev/kpt/releases)

```

**Comment by electrocucaracha** [comment]
@lapentad This installation method is not exclusive for OpenStack, I think that it needs to be restructured to avoid any confusion.

**Comment by lapentad** [comment]
> @lapentad This installation method is not exclusive for OpenStack, I think that it needs to be restructured to avoid any confusion.

I'll change the filename and titles 
/hold

**Pull request merged**